plotly_resampler/figure_resampler/figure_resampler_interface.py

```diff
@@ -1302,23 +1302,57 @@ def _parse_relayout(self, relayout_dict: dict) -> dict:
             extra_layout_updates[f"{axis}.autorange"] = None
         return extra_layout_updates
 
-    def construct_update_data(
```

Comment:
Good work guys, be aware this changes the interface. When it's not backwards compatible it should be tagged with a major version bump if following Semantic Versioning. Congrats on 1.0.0 :D

Reply:
Would be nice to add this to the change log, with supporting documentation to use the new interface. This broke my code and I had to dig to find this change.